### DISCUSSION TITLE
How to get to work with NextJS?

### DISCUSSION THREAD

**Discussion by jamezening** [themesberg/flowbite#85] [Q&A] [answered]
Has anyone been able to get flowbite to work with NextJS?
I m following the steps here:
https://flowbite.com/docs/getting-started/react/#flowbite-components

and get a 
```
Server Error
ReferenceError: document is not defined

This error happened while generating the page. Any console logs will be displayed in the terminal window.
```

after adding `import 'flowbite';` to my pages/index.tsx.

Also tried the Next Script way
`<Script src="..../node_modules/flowbite" />`
but can't get the js functionality to work.

Thank you for any guidance!

**Comment by stefanovolpatti** [comment, marked as answer]
Hi @gyoon-dev 

An issue has already been opened and there are some answers about this topic.
You can find the issue [here](https://github.com/themesberg/flowbite/issues/51)

**Reply by jamezening** [reply]
@volp99 
Thank you for the heads up. Will track the issue there.
